docs/ref/templates/builtins.txt

```diff
@@ -1422,6 +1422,15 @@ Format character  Description                               Example output
                   (January 1 1970 00:00:00 UTC).
 ================  ========================================  =====================
 
+Note that you can backslash-escape a format string if you want to use the
+"raw" value. In this example, both "o" and "f" are backslash-escaped, because
+otherwise each is a format string that displays the year and the time,
+respectively::
+
+    It is the {% now "jS \o\f F" %}
+
+This would display as "It is the 4th of September".
+
```

Comment:
I don't think that we need a separate example in the `date` docs. I would rather add a short note to [the existing example](https://github.com/django/django/pull/12128/files#diff-7be9aaef6dad344e74188264c0e95daaR1461-R1462) for `es`, e.g.
```
...
outputs ``9 de Enero de 2008`` (the ``DATE_FORMAT`` format specifier for the
``es`` locale is ``r'j \d\e F \d\e Y'``). Both "d" and "e" are
backslash-escaped, because otherwise each is a format string that displays the
day and the timezone name, respectively.
```

Reply:
OK. I think I see what you're saying. I'll try again over the holiday weekend. Thanks!

Reply:
@felixxm I've made the change suggested. Thanks!